dask/show_versions.py

```diff
@@ -0,0 +1,57 @@
+from json import dumps
+from platform import uname
+from sys import stdout, version_info
+
+from pandas._typing import JSONSerializable
+from pandas.compat._optional import get_version, import_optional_dependency
```

Comment:
I'd prefer to avoid using private methods where possible as they may go away / have breaking changes without warning